services/src/main/java/edu/unc/lib/dl/cdr/services/rest/DatastreamRestController.java

```diff
@@ -17,7 +17,9 @@
 
 import static edu.unc.lib.dl.acl.util.GroupsThreadStore.getAgentPrincipals;
 import static edu.unc.lib.dl.fcrepo4.RepositoryPathConstants.ORIGINAL_FILE;
+import static edu.unc.lib.dl.util.DerivativeService.isDerivative;
```

Comment:
i'm not going to draw a line in the sand or anything, but would you consider naming this class "DatastreamController"? we've moved away from including the word "rest" in our other newer controllers :-)

Reply:
Sure, but this is a pre-migration class and there are 3 other classes in the same package with Rest in their name